CRM/Event/Form/ManageEvent.php

```diff
@@ -378,7 +366,7 @@ public function endPostProcess() {
 
       CRM_Core_Session::setStatus(ts("'%1' information has been saved.",
         [1 => CRM_Utils_Array::value('title', CRM_Utils_Array::value($subPage, $this->get('tabHeader')), $className)]
-      ), ts('Saved'), 'success');
+      ), $this->getVar('_title'), 'success');
```

Comment:
A getTitle() fn would be nicer...

Reply:
& isn't 'Saved' OK anyway?

Reply:
And there's already a `getTitle()` function :-)  Now updated to use that - it gets a bit tedious when working with multiple events to see "Saved" as the title when we can easily use the event title.